crates/isograph_cli/src/main.rs

```diff
@@ -60,13 +55,27 @@ async fn start_language_server(lsp_command: LspCommand) {
             .config
             .unwrap_or("./isograph.config.json".into()),
     );
-    eprintln!("Starting language server");
+    info!("Starting language server");
     if let Err(_e) = isograph_lsp::start_language_server(config).await {
-        eprintln!(
+        error!(
             "{}",
             "Error encountered when running language server.".bright_red(),
             // TODO derive Error and print e
         );
         std::process::exit(1);
     }
 }
+
+// TODO: Configure parameters based on provided log_level.
```

Comment:
what is this TODO referring to? This looks like it is using log_level

Reply:
That is for the future me :) I want to provide different tracing parameters, like function name, line number, etc. for debug and trace levels. But I can add it in this PR as well.

Reply:
sgtm